front/app/components/CustomFieldsForm/Page/PageTitle.tsx

```diff
@@ -13,29 +13,37 @@ const PageTitle = ({ page }: { page: IFlatCustomField }) => {
   const localize = useLocalize();
   const isMobileOrSmaller = useBreakpoint('phone');
   const theme = useTheme();
+
+  const title = localize(page.title_multiloc);
+  const description = localize(page.description_multiloc);
```

Comment:
Kind of unrelated, but now we only render the title and description components if the title / description are there